### PR TITLE
Fix force close when Questionnaire launch fails with exception

### LINKED ISSUE #2405: App crash when you launch questionnaires
**Describe the bug**
App crashes when you launch questionnaires. 

**To Reproduce**
1. Go to Household register 
2. Open any household profile
3. Click Add Member fab
4. Application force closes

**Expected behavior**
Launching a questionnaire should not crash the application whether initial expression is used or not.

**Additional context**
Exception observed after the fix to use initial expressions on questionnaires. This affects majority of the questionnaires.

### PR DESCRIPTION
**Pull request by ellykits**
**IMPORTANT: Where possible all PRs must be linked to a Github issue**

Fixes #2405

**Engineer Checklist**
- [x] I have written **Unit tests** for any new feature(s) and edge cases for bug fixes
- [x] I have added any strings visible on UI components to the `strings.xml` file
- [ ] I have updated the  [CHANGELOG.md](./CHANGELOG.md) file for any notable changes to the codebase
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the project's style guide
- [x] I have built and run the FHIRCore app to verify my change fixes the issue and/or does not break the app 

**Code Reviewer Checklist**
- [ ] I have verified **Unit tests** have been written for any new feature(s) and edge cases
- [ ] I have verified any strings visible on UI components are in the `strings.xml` file
- [ ] I have verifed the [CHANGELOG.md](./CHANGELOG.md) file has any notable changes to the codebase
- [ ] I have verified the solution has been implemented in a configurable and generic way for reuseable components
- [ ] I have built and run the FHIRCore app to verify the change fixes the issue and/or does not break the app

### CODE REVIEW

**Review by dubdabasoduba — APPROVED** [review]

File: android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt [hunk 1]
```diff
@@ -147,20 +146,27 @@ open class QuestionnaireActivity : BaseMultiLanguageActivity(), View.OnClickList
...
-          questionnaireViewModel.getQuestionnaireResponseFromDbOrPopulation(
-              questionnaire = questionnaire,
-              subjectId = baseResourceId.extractLogicalIdUuid(),
-              subjectType = baseResourceType,
-              questionnaireConfig = questionnaireConfig
-            )
-            .apply { generateMissingItems(questionnaire) }
+          if ((baseResourceId.isNullOrEmpty() || baseResourceType == null) &&
+              questionnaireConfig.type.isDefault()
+          ) {
```
Comment (aurangzaibumer) [inline android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt:151]: @dubdabasoduba I think this is why you are unable to load expressions, try giving questionnareType.Edit in your case

Comment (ellykits) [reply android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireActivity.kt:151]: I see

**Comment by pld** [comment]
is there anything we can test around this?

**Comment by ellykits** [comment]
There are tests in the `QuestionnaireViewModel` for the affected areas of code.

**Pull request merged**